net/restic-rest-server/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=restic-rest-server
+PKG_VERSION:=0.9.7
```

Comment:
restic/rest-server@a87d968870b4dec64abfe1270e538fdc01524a81 is actually tagged `0.9.8` (even though the VERSION file still says `0.9.7`). Do you want to package 0.9.7 or 0.9.8?

Reply:
Sorry, 0.9.7. 0.9.8 seems somewhat incomplete to me.